Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc

```diff
@@ -132,6 +132,7 @@ export function install(args: Arguments) : Result {
                             d`${Context.getMount("LocalAppData").path}/yarn`,
                         ]),
                     ],
+                    requireCbDependencies: true,
                 },

```

Comment:
Why does yarn require cb dependencies? I thought we were only using this for building the explorer.

Reply:
yes, it is used for building the explorer.
node.exe will access the nuget credential provider.